src/events/aws_sdk_v3.js

```diff
@@ -56,6 +57,71 @@ const SNSv3EventCreator = {
     },
 };
 
+const SQSEventCreator = {
+    /**
+     * Updates an event with the appropriate fields from a SQS command
+     * @param {string} operation the operation we wrapped.
+     * @param {Command} command the wrapped command
+     * @param {proto.event_pb.Event} event The event to update the data on
+     */
+    requestHandler(operation, command, event) {
+        const parameters = command.input || {};
+        const resource = event.getResource();
+
+        if ('QueueUrl' in parameters) {
+            resource.setName(`${parameters.QueueUrl.split('/').pop()}`);
+        }
+
+        const entry = ('Entries' in parameters) ? parameters.Entries : parameters;
+        if ('MessageBody' in entry) {
+            eventInterface.addToMetadata(event, {}, {
+                'Message Body': entry.MessageBody,
+            });
+        }
+
+        if ('MessageAttributes' in entry) {
+            eventInterface.addToMetadata(event, {}, {
+                'Message Attributes': entry.MessageAttributes,
+            });
+        }
+    },
+
+    /**
+     * Updates an event with the appropriate fields from a SQS response
+     * @param {string} operation the operation we wrapped.
+     * @param {object} response The AWS.Response object
+     * @param {proto.event_pb.Event} event The event to update the data on
+     */
+    responseHandler(operation, response, event) {
+        switch (operation) {
+        case 'SendMessageCommand':
+            eventInterface.addToMetadata(event, {
+                'Message ID': `${response.MessageId}`,
+                'MD5 Of Message Body': `${response.MD5OfMessageBody}`,
```

Comment:
Is that the convention for MD5 tag?
Verify this

Reply:
Yes, verified it